alpha/lib/model/category.php

```diff
@@ -1772,8 +1782,10 @@ public function indexCategoryInheritedTree()
 	 * @param $fullIds
 	 * @return array
 	 */
-	private static function getCategoriesStartWithIds($fullIds)
+	private function getDescendantCategoriesIds()
 	{
+		$fullIds = $this->getFullIds();
+		$fullIds = $fullIds.'>';
 		if (!isset($fullIds) || empty($fullIds))
```

Comment:
This validation is redundant now that you build the full-ids yourself and not accpeting them as method input.